### PR TITLE
Fix docs not correctly building for some utility APIs

### PR DESCRIPTION
**Pull request by cjohnston1158**
This is a first pass to get most of the utility API docs working properly again.

### CODE REVIEW

File: zaza/utilities/cert.py [hunk 1]
```diff
@@ -190,7 +190,7 @@ def sign_csr(csr, ca_private_key, ca_cert=None, issuer_name=None,
         datetime.datetime.today() - datetime.timedelta(1, 0, 0),
     )
     builder = builder.not_valid_after(
-        datetime.datetime.today() + datetime.timedelta(80, 0, 0),
+        datetime.datetime.today() + datetime.timedelta(4000, 0, 0),
```
Comment (thedac) [inline zaza/utilities/cert.py]: I suspect this is unrelated. Can we break this out to a separate change?

File: zaza/model.py [hunk 2]
```diff
@@ -359,6 +359,7 @@ def get_unit_from_name(unit_name, model=None, model_name=None):
 
     If pgrep_full is True  ensure that any special characters in the name of
     the service are escaped e.g.
+
```
Comment (cjohnston1158) [inline zaza/model.py]: This was done to fix a build error in the docs.
